contracts/service-registry/src/state.rs

```diff
@@ -89,7 +89,7 @@ impl BondingState {
             | BondingState::Unbonding {
                 amount,
                 unbonded_at: _,
-            } => amount + to_add,
+            } => amount.saturating_add(to_add),
```

Comment:
very dangerous, because we are taking one amount for staking that's being sent through the bank module and potentially set a different amount here. This must be `checked_add()?`